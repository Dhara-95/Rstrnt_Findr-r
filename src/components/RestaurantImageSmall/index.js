import React from "react";

import { checkValidImageURL } from "../../utils/checkValidImageURL";

const RestaurantImageSmall = (props) => {
  const { url, name } = props;

  return <img src={checkValidImageURL(url)} width="150" alt={name} />;
};

export { RestaurantImageSmall };
