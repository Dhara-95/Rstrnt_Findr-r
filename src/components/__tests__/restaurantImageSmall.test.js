import React from "react";
import renderer from "react-test-renderer";

import { RestaurantImageSmall } from "../RestaurantImageSmall";

it("displays the Restaurant name correctly", () => {
  const tree = renderer
    .create(<RestaurantImageSmall url={"url"} name={"name"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
