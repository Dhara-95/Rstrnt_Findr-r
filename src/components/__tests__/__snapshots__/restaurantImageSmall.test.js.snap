// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Restaurant name correctly 1`] = `
<div
  className="flex w-48 h-48 rounded-3xl overflow-hidden relative justify-center content-center "
>
  <img
    alt="name"
    className="w-auto h-full block object-cover transform scale-130"
    src="url"
  />
</div>
`;
